setup.py

```diff
@@ -54,6 +54,8 @@
     "dockerctx",
     "pyarrow",
     "psycopg2-binary",
+    "psycopg-binary",
+    "psycopg-pool",
```

Comment:
We added it for debugging when we thought it was questdb. I think we can keep it as it was before.

Reply:
Got it!

Reply:
fix in [1ae8673](https://github.com/ploomber/jupysql/pull/493/commits/1ae867365472eedb110e4bbf9ee42f7f5ba53a97)